gulpfile.js

```diff
@@ -54,13 +54,13 @@ gulp.task('clean-release', clean_release);
 
 gulp.task('clean-cache', clean_cache);
 
-var distBuild = gulp.series(clean_dist, dist);
-gulp.task('dist', distBuild);
+var distRebuild = gulp.series(clean_dist, dist);
+gulp.task('dist', distRebuild);
 
-var appsBuild = gulp.series(gulp.parallel(clean_apps, distBuild), apps, gulp.parallel(listPostBuildTasks(APPS_DIR)));
+var appsBuild = gulp.series(gulp.parallel(clean_apps, distRebuild), apps, gulp.parallel(listPostBuildTasks(APPS_DIR)));
 gulp.task('apps', appsBuild);
 
-var debugBuild = gulp.series(gulp.parallel(clean_debug, distBuild), debug, gulp.parallel(listPostBuildTasks(DEBUG_DIR)), start_debug)
+var debugBuild = gulp.series(gulp.parallel(clean_debug, dist), debug, gulp.parallel(listPostBuildTasks(DEBUG_DIR)), start_debug)
```

Comment:
I think we should consider removing `clean_debug` from this step as well - this will give us even more reduction in the rebuild time when debug builds are used, which is (as far as I can tell) the aim of this change.

Reply:
Yes, you are right. Fixed.